src/pages/imprint.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { HelmetSeo } from '~/components/Helmet/HelmetSeo';
+import { LayoutArticle } from '~/components/Layout';
+import { ExternalLink } from '~/components/Links/ExternalLink';
+import BmdvFundingImage from '~/static/BMDV_Foerderung.svg';
+
+const ImprintPage = () => {
+  return (
+    <LayoutArticle>
+      <HelmetSeo noindex title="Impressum" />
```

Comment:
I did not create an imprint page by design but a contact page.
We would have only one of the two, since the contents of the page are very redundant. 
And I suggest to use the /contact route since that is the nicer name for the main intend of the page – getting to know and contacting the owner.

Reply:
As I know, it is mandatory in Germany to have the page called "Impressum" and is visible on the first page.

Reply:
The page can still be a contact page primarily and have an imprint section. The URL can also be contact and the primary menu as well.

However, I agree the save route is to also use the term "Impressum" eg in the footer (and link to the same page).

It's not required to have two pages with nearly the same content, though. 
Example: https://www.betterplace.org/c/kontakt
(Footer and Page)